Cargo.toml

```
@@ -228,6 +228,7 @@ members = [
 	"utils/frame/rpc/system",
 	"utils/frame/generate-bags",
 	"utils/frame/generate-bags/node-runtime",
+	"utils/frame/rpc-utils",
```

```suggestion
	"utils/frame/rpc",
```

we're already in a folder called `utils`, sounds not needed to repeat it.